node/src/components/consensus/tests/mock_proto.rs

```diff
@@ -0,0 +1,253 @@
+use std::{
+    any::Any,
+    collections::{BTreeSet, HashSet, VecDeque},
+};
+
+use casper_execution_engine::shared::motes::Motes;
+use datasize::DataSize;
+use derive_more::Display;
+use serde::{de::DeserializeOwned, Deserialize, Serialize};
+
+use crate::{
+    components::{
+        chainspec_loader::Chainspec,
+        consensus::{
+            consensus_protocol::{
+                BlockContext, ConsensusProtocol, ConsensusProtocolResult as CpResult,
```

Comment:
I see why you changed `ConsensusProtocolResult` -> `CpResult` in the file above.  But it's not clear we need that change was part of this feature.